src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -0,0 +1,221 @@
+        issuingAuthority:
+          type: string
+          maxLength: 255
+          description: Issuing authority (LA/ERO/VJB)
+          example: Camden Borough Council
+        issueDate:
+          type: string
+          format: date
+          description: The issue date of the Voter Authority Card
+          example: '2022-06-01'
+        requestDateTime:
+          type: string
+          format: date-time
+          description: The date time the Voter Authority Certificate was requested
+          example: '2022-06-01T12:23:03.000Z'
+        cardFirstname:
+          type: string
+          maxLength: 255
+          description: First name of the Elector
+          example: John
+        cardMiddlenames:
+          type: string
+          maxLength: 255
+          description: Middle names of the Elector
+          example: Malcolm
+        cardSurname:
+          type: string
+          maxLength: 255
+          description: Surname of the Elector
+          example: Smith
+        certificateLanguage:
+          $ref: '#/components/schemas/CertificateLanguage'
+        deliveryOption:
+          $ref: '#/components/schemas/DeliveryOption'
+        photoS3Arn:
+          type: string
+          description: S3 ARN to image to send to printers.
+          maxLength: 1024
+        deliveryName:
+          type: string
+          maxLength: 255
+          description: Name to be printed with delivery address
+        deliveryAddress:
+          $ref: '#/components/schemas/Address'
+        eroEnglish:
+          $ref: '#/components/schemas/ElectoralRegistrationOffice'
+        eroWelsh:
+          $ref: '#/components/schemas/ElectoralRegistrationOffice'
+      required:
+        - voterApplicationId
+        - applicationReference
+        - issuingAuthority
+        - issueDate
+        - requestDateTime
+        - cardFirstname
+        - cardMiddlenames
+        - cardSurname
+        - certificateLanguage
+        - deliveryOption
+        - photoS3Arn
+        - deliveryName
+        - deliveryAddress
+        - eroEnglish
+
+    CertificateLanguage:
+      title: CertificateLanguage
+      description: The Voter Authority Certificate language
+      type: string
+      enum:
+        - cy
+        - en
+      default: en
+
+    DeliveryOption:
+      title: DeliveryOption
+      description: The delivery option for the Voter Authority Certificate
+      type: string
+      enum:
+        - standard
+
+    Address:
+      title: Address
+      description: Address format used throughout gov.uk voter services.
+      type: object
+      x-examples:
+        Minimum data:
+          street: Street 1
+          postcode: PC1 2FB
+        Normal Address:
+          street: East Lodge
+          property: Balruddery
+          locality: Invergowrie
+          town: Dundee
+          area: Angus
+          postcode: DD25LF
+          uprn: '117095813'
```

Comment:
@NathanRussellValtech , uprn is not defined as a property for the Address object. Do we need to define it as well for the Address, or is this example not correct?